contrib/babelfishpg_tds/src/backend/tds/tds.c

```diff
@@ -395,7 +406,17 @@ tdsstat_bestart(void)
 	ltdsentry.textsize = atoi(GetConfigOption("babelfishpg_tsql.textsize", true, true));
 	ltdsentry.datefirst = atoi(GetConfigOption("babelfishpg_tsql.datefirst", true, true));
 	ltdsentry.lock_timeout = atoi(GetConfigOption("lock_timeout", true, true));
-	ltdsentry.transaction_isolation = atoi(GetConfigOption("default_transaction_isolation", true, true));
+
+	isolation_level_str = GetConfigOption("default_transaction_isolation", true, true);
```

Comment:
can't we use DefaultXactIsoLevel here, that way we do not need to do all the conversion?